packages/grafana-data/src/dataframe/frameComparisons.ts

```diff
@@ -6,7 +6,7 @@ import { DataFrame } from '../types/dataFrame';
  *
  * To compare multiple frames use:
  * ```
- * areArraysEqual(a, b, framesHaveSameStructure);
+ * compareArrayValues(a, b, framesHaveSameStructure);
```

Comment:
unrelated... but I noticed it while working on this PR

Reply:
👍